service-registrar/src/main/java/com/msm/sr/service/status/notification/manager/ServiceStatusNotifierManager.java

```diff
@@ -8,10 +8,13 @@
  */
 public class ServiceStatusNotifierManager {
 
-    private static final ServiceStatusNotifier SERVICE_STATUS_NOTIFIER = new ServiceStatusNotifier();
+    private static final ServiceStatusNotifier serviceStatusNotifier = new ServiceStatusNotifier();
+
+    private ServiceStatusNotifierManager() {
```

Comment:
Why do you need the default constructor for? `ServiceStatusNotifierManager` only contains static methods so there is no need for creating the constructor. Also Java classes by default have the default constructor so there is really no need for declaring it.